sql/moz-fx-data-marketing-prod/google_search_console_derived/search_impressions_by_page_v2/metadata.yaml

```diff
@@ -21,9 +21,22 @@ scheduling:
   date_partition_parameter: date
   # Google Search Console exports typically happen two days after the data date in UTC.
```

Comment:
Would be nice to have this added directly as a comment in the generated DAG

Reply:
An equivalent comment is [in the DAG description](https://github.com/mozilla/bigquery-etl/blob/cfd16aaef47dacdbc802683f2b81d9a9d66ec1d1/dags.yaml#L1480), which ends up in the DAG code as a docstring and is visible in the Airflow UI.